android/common/src/main/java/org/kotlinacademy/common/ContextExt.kt

```diff
@@ -11,18 +11,17 @@ val Context.notificationManager: NotificationManager
 fun Context.openUrl(url: String?) {
     url.nullIfBlank() ?: return
     val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-    if (browserIntent.resolveActivity(packageManager) != null) {
-        startActivity(browserIntent)
-    }
+    browserIntent.resolveActivity(packageManager).let { startActivity(browserIntent) }
 }
 
 fun Context.startShareIntent(subject: String, text: String) {
-    val intent = Intent(Intent.ACTION_SEND).apply {
```

Comment:
Same story - additional complexity and no real improvement.

Reply:
Ok--will revert this change